Coupling LAMMPS (examples/COUPLE/simple)

**Summary**

1. git clone https://github.com/lammps/lammps.git lammps
2. `make mpi mode=shlib`
3. Now I want to use lammps in my other C++ project, like explained in `examples/COUPLE/simple/`, , i.e. I do: `#include "library.h"`.
4. Observe error `#elif with no expression`.

**LAMMPS Version and Platform**

LAMMPS head of master from Github. GCC8 on Ubuntu Linux Xenial AMD64.

**Details**

Error I'm facing:
```
In file included from /builds/stephanmg/coupling/simulator_interface.h:25:0,
/builds/stephanmg/lammps/src/library.h:64:22: error: #elif with no expression
 #elif LAMMPS_SMALLBIG
```

Hope this is a issue to be reported, because I'm including LAMMPS from another project as described in the couple example.

--- a/src/library.h
+++ b/src/library.h
@@ -16,8 +16,12 @@
    new LAMMPS-specific functions can be added
 */
 
+/*
+ * Follow the behavior of regular LAMMPS compilation and assume
+ * -DLAMMPS_SMALLBIG when no define is set.
+ */
 #if !defined(LAMMPS_BIGBIG) && !defined(LAMMPS_SMALLBIG) && !defined(LAMMPS_SMALLSMALL)
-#error Must define LAMMPS_BIGBIG, LAMMPS_SMALLBIG, or LAMMPS_SMALLSMALL
+#define LAMMPS_SMALLBIG
 #endif
 
 #include <mpi.h>

--- a/src/library.h
+++ b/src/library.h
@@ -16,6 +16,10 @@
    new LAMMPS-specific functions can be added
 */
 
+#if !defined(LAMMPS_BIGBIG) && !defined(LAMMPS_SMALLBIG) && !defined(LAMMPS_SMALLSMALL)
+#error Must define LAMMPS_BIGBIG, LAMMPS_SMALLBIG, or LAMMPS_SMALLSMALL
+#endif
+
 #include <mpi.h>
 #if defined(LAMMPS_BIGBIG) || defined(LAMMPS_SMALLBIG)
 #include <inttypes.h>  /* for int64_t */
@@ -58,10 +62,10 @@ void lammps_gather_atoms_subset(void *, char *, int, int, int, int *, void *);
 void lammps_scatter_atoms(void *, char *, int, int, void *);
 void lammps_scatter_atoms_subset(void *, char *, int, int, int, int *, void *);
 
-#ifdef LAMMPS_BIGBIG
+#if defined(LAMMPS_BIGBIG)
 typedef void (*FixExternalFnPtr)(void *, int64_t, int, int64_t *, double **, double **);
 void lammps_set_fix_external_callback(void *, char *, FixExternalFnPtr, void*);
-#elif LAMMPS_SMALLBIG
+#elif defined(LAMMPS_SMALLBIG)
 typedef void (*FixExternalFnPtr)(void *, int64_t, int, int *, double **, double **);
 void lammps_set_fix_external_callback(void *, char *, FixExternalFnPtr, void*);
 #else

